easybuild/easyblocks/g/gamess_us.py

```diff
@@ -0,0 +1,232 @@
+            tup = (known_ddi_comms, self.cfg['ddi_comm'])
+            self.log.error("Unsupported DDI communication layer specified (known: %s): %s" % tup)
+
+        # MPI library config
+        mpilib, mpilib_root, mpilib_path = None, None, None
+        if self.cfg['ddi_comm'] == 'mpi':
+
+            known_mpilibs = ['impi', 'OpenMPI', 'MVAPICH2', 'MPICH2']
+            for mpilib in known_mpilibs:
+                mpilib_root = get_software_root(mpilib)
+                if mpilib_root is not None:
+                    break
+            if mpilib_root is None:
+                self.log.error("None of the known MPI libraries (%s) available, giving up." % known_mpilibs)
+            mpilib_path = mpilib_root
+            if mpilib == 'impi':
+                mpilib_path = os.path.join(mpilib_root, 'intel64')
+
+        # run interactive 'config' script to generate install.info file
+        cmd = "%(preconfigopts)s ./config %(configopts)s" % {
+            'preconfigopts': self.cfg['preconfigopts'],
+            'configopts': self.cfg['configopts'],
+        }
+        qa = {
+            "After the new window is open, please hit <return> to go on.": '',
+            "please enter your target machine name: ": machinetype,
+            "Version? [00] ": self.version,
+            "Please enter your choice of FORTRAN: ": fortran_comp,
+            "Enter only the main version number, such as 8, ... 11, 12, 13, 14.\nVersion? ": fortran_ver,
+            "hit <return> to continue to the math library setup.": '',
+            "Enter your choice of 'mkl' or 'atlas' or 'acml' or 'none': ": mathlib,
+            "MKL pathname? ": mathlib_root,
+            "MKL version (or 'skip')? ": 'skip',
+            "please hit <return> to compile the GAMESS source code activator": '',
+            "please hit <return> to set up your network for Linux clusters.": '',
+            "communication library ('sockets' or 'mpi')? ": self.cfg['ddi_comm'],
+            "Enter MPI library (impi, mvapich2, mpt, sockets):": mpilib,
+            "Please enter your %s's location: " % mpilib: mpilib_root,
+            "Do you want to try LIBCCHEM?  (yes/no): ": 'no',
+        }
+        stdqa = {
+            r"GAMESS directory\? \[.*\] ": self.builddir,
+            r"GAMESS build directory\? \[.*\] ": self.installdir,  # building in install directory
+        }
+        run_cmd_qa(cmd, qa=qa, std_qa=stdqa, log_all=True, simple=True)
+
+        self.log.debug("Contents of install.info:\n%s" % read_file(os.path.join(self.builddir, 'install.info')))
+
+        # patch hardcoded settings in rungms to use values specified in easyconfig file
+        rungms = os.path.join(self.builddir, 'rungms')
+        try:
+            for line in fileinput.input(rungms, inplace=1, backup='.orig'):
+                line = re.sub(r"^(\s*set\s*TARGET)=.*", r"\1=%s" % self.cfg['ddi_comm'], line)
+                line = re.sub(r"^(\s*set\s*GMSPATH)=.*", r"\1=%s" % self.installdir, line)
```

Comment:
TODO: also define `ERICFMT ` and `MCPPATH `, see https://github.com/hpcugent/easybuild-easyconfigs/pull/1352/files#diff-e899eb0decac3e9b1edfaf407ba28764R29